website/content/gateway/api_reference/transaction_status/transaction_status_codes_table.md

```diff
@@ -4,27 +4,27 @@ date: 2022-04-13T12:37:22+02:00
-| -------- | ----- | ---------------------------------- | ---- | ------- | ------ | ---- | ------ |
-| Approved | 20000 | Approved                           | ✓    | ✓       | ✓      | ✓    | ✓      |
-| Declined | 40000 | General input error                | ✓    | ✓       | ✓      | ✓    | ✓      |
-|          | 40110 | Invalid card number                | ✓    |         |        |      | ✓      |
-|          | 40111 | Unsupported card scheme            | ✓    |         |        |      | ✓      |
-|          | 40120 | Invalid CSC                        | ✓    |         |        |      |        |
-|          | 40130 | Invalid expire date                | ✓    |         |        |      | ✓      |
-|          | 40135 | Card expired                       | ✓    |         |        |      | ✓      |
-|          | 40140 | Invalid currency                   | ✓    |         |        |      | ✓      |
-|          | 40150 | Invalid text on statement          | ✓    | ✓       | ✓      |      | ✓      |
-|          | 40200 | Clearhaus rule violation           | ✓    | ✓       | ✓      | ✓    | ✓      |
-|          | 40300 | 3-D Secure problem                 | ✓    |         |        |      |        |
-|          | 40310 | 3-D Secure authentication failure  | ✓    |         |        |      |        |
-|          | 40400 | Backend problem                    | ✓    |         |        | ✓    | ✓      |
-|          | 40410 | Declined by issuer or card scheme  | ✓    |         |        | ✓    | ✓      |
-|          | 40411 | Card restricted                    | ✓    |         |        |      | ✓      |
-|          | 40412 | Card lost or stolen                | ✓    |         |        |      | ✓      |
-|          | 40413 | Insufficient funds                 | ✓    |         |        |      | ✓      |
-|          | 40414 | Suspected fraud                    | ✓    |         |        |      | ✓      |
-|          | 40415 | Amount limit exceeded              | ✓    |         |        |      | ✓      |
-|          | 40416 | Additional authentication required | ✓    |         |        |      | ✓      |
-|          | 40420 | Merchant blocked by cardholder     | ✓    |         |        |      | ✓      |
-|          | 50000 | Clearhaus error                    | ✓    | ✓       | ✓      | ✓    | ✓      |
+| Status   | code  | Meaning                            | Auth | Capture | Refund | Void | Credit | Debit  | Debit Refund |
+|----------|-------|------------------------------------|------|---------|--------|------|--------|--------|--------------|
+| Approved | 20000 | Approved                           | ✓    | ✓       | ✓      | ✓    | ✓      | ✓      | ✓            |
+| Declined | 40000 | General input error                | ✓    | ✓       | ✓      | ✓    | ✓      | ✓      | ✓            |
+|          | 40110 | Invalid card number                | ✓    |         |        |      | ✓      | ✓      |              |
```

Comment:
```suggestion
|          | 40110 | Invalid card number                | ✓    |         |        |      | ✓      | ✓      | ✓            |
```

It sounds obscure, however, we cannot guarantee that we will not respond with invalid card number for a debit refund. We are talking to upstream, i.e. Visa, and an issuer, or Visa, might respond to us with ISO 8583 response code 14 meaning invalid card number.

It is quite unrealistic, though, as a debit refund must happen relatively quickly after a debit.

@ct-clearhaus WDYT?

Reply:
Issuers can potentially send us any ISO 8583 response code, and we would do the usual mapping expected of us.

Reply:
Looking at the current 404xx status codes for debit refunds, it seems we lean towards putting a checkmark on all possible responses from upstream. So I'm leaning towards we do that consistently for all possible responses (i.e. also for invalid card number, invalid CSC, card expired, merchant blocked by cardholder).

However, it does feel weird to mention invalid CSC as a possibility for a debit refund.

Reply:
I've striked "merchant blocked by cardholder" above as I see that's not a possible case for debits and debit refunds.

Reply:
The debit refund is a "live" transaction, so more response codes are actually possible, yes 👍

I didn't think about this 🙈